macros/unpack/get_metrics.sql

```diff
@@ -36,13 +43,14 @@
               'file_path', 
               'is_described', 
               'metric_type', 
-              'model', 
```

Comment:
any reason to remove this field? I think this is failing in a downstream model! See the CircleCI error, i think it's referenced in `int_all_graph_resources`

Reply:
AH - got it, was wondering where that failure was coming from. I guess this depends on whether we want to include blank fields for this. We're going to be adding `expression` metrics soon that will be dependent on parent metrics instead of models and so the model information in the node will be empty.

If its got relevance to downstream models used in this package, we can definitely add it back in and just populate a blank string value.

Reply:
I've re-added it back in to see if it passes integration tests. The `model` attribute of the `metric` node doesn't have an identifier aspect so this will always return a blank value.